packages/calcite-components/src/components/loader/loader.scss

```diff
@@ -3,10 +3,17 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-loader-font-size: Specifies the font size of the loading percentage when type is `"determinate"`.
- * @prop --calcite-loader-size: The width and height of a non-inline loader.
- * @prop --calcite-loader-size-inline: The width and height of an inline loader
- * @prop --calcite-loader-padding : Specifies the padding of the loader.
+ * @prop --calcite-loader-color-oneThird: defines the color of a loader. Applies to --calcite-loader-color between 33% - 66% of the animation.
+ * @prop --calcite-loader-color-start:  defines the starting color of a loader. Applies to --calcite-loader-color between 0% - 33% of the animation.
+ * @prop --calcite-loader-color-twoThirds: defines the color of a loader. Applies to --calcite-loader-color between 66% - 99% of the animation.
+ * @prop --calcite-loader-bar-width:  *deprecated* defines the width of the loader animation. 
+ * @prop --calcite-loader-font-size: *deprecated* Specifies the font size of the loading percentage when type is `"determinate"`.
+ * @prop --calcite-loader-padding: *deprecated* Specifies the padding of the loader.
```

Comment:
I think we need to use `[Deprecated]` instead of `*deprecated*` right @macandcheese ?

Might need to search and cleanup if used elsewhere.